src/Microsoft.Extensions.DiagnosticAdapter/DiagnosticSourceAdapter.cs

```diff
@@ -79,7 +125,7 @@ public void Write(string diagnosticName, object parameters)
                 return;
             }
 
-            if (_listener.IsEnabled != null && !_listener.IsEnabled(diagnosticName))
+            if (_listener.IsEnabled != null && !_listener.IsEnabled(diagnosticName, null, null))
```

Comment:
So if we just always call this with `null`, `null` then we're not really supporting it. Where are we supposed to get these values from?